src/frontend/hooks/persistedState/usePersistedDraftObservation/index.ts

```diff
@@ -76,6 +76,7 @@ const draftObservationSlice: StateCreator<DraftObservationSlice> = (
           metadata: {
             ...prevValue.metadata,
             position: props.position,
+            manualLocation: props.manualLocation,
```

Comment:
super subtle but found out why `updateObservationPosition()` didn't update the `manualLocation` field when initially implementing this 🥲